autofocus/predict/app/requests/Validator.py

```diff
@@ -0,0 +1,63 @@
+from abc import ABC, abstractmethod
+import mimetypes
+
+from flask import abort, jsonify, make_response
+from flask_api import status
+
+
+ALLOWED_IMAGE_FILES = set(
+    k for k, v in mimetypes.types_map.items() if v.startswith("image/")
+)
+ALLOWED_ZIP_FILES = {".zip"}
+
+
+class Validator(ABC):
```

Comment:
The class structure here seems to me to add more overhead than it is worth. All we need is one function to validate `predict` requests, one to validate `predict_multiple` requests, and appropriate calls to abort.

Reply:
I see your point and changed this to be a single method now.